derive/src/util.rs

```diff
@@ -528,6 +540,7 @@ pub(crate) fn get_func_sig(sig: &Signature, name: &str) -> String {
             let ident = match arg.pat.as_ref() {
                 syn::Pat::Ident(p) => p.ident.to_string(),
                 // FIXME: other => unreachable!("function arg pattern must be ident but found `{}`", quote!(fn #ident(.. #other ..))),
+                // FIXME: Pat::PatStruct should be captured though and added to sig. (Common destructuring with custom Args)
```

Comment:
I think FromArgs trait need to have something like `FromArgs::text_signature()` to describe args inside of it. PatStruct will not cover when FromArgs struct is used as a single argument, but trait static method will cover both cases.

Reply:
I'll look into that, dropped the comment for now.